pkg/ksym/ksym.go

```
@@ -236,12 +250,8 @@ func (c *cache) resolveKsyms(addrs []uint64) []string {
 	result := make([]string, 0, len(addrs))
 
 	for _, addr := range addrs {
-		idx := sort.Search(len(c.kernelSymbols), func(i int) bool { return addr < c.kernelSymbols[i].address })
-		if idx < len(c.kernelSymbols) && idx > 0 {
-			result = append(result, c.kernelSymbols[idx-1].name)
-		} else {
-			result = append(result, "")
-		}
+		name, _ := c.searcher.Search(addr)
```

Let's ignore the error.